guides/source/webpacker.md

```diff
@@ -177,18 +179,15 @@ As of Webpacker version 5, Webpacker is not "engine-aware," which means Webpacke
 
 Gem authors of Rails engines who wish to support consumers using Webpacker are encouraged to distribute frontend assets as an NPM package in addition to the gem itself and provide instructions (or an installer) to demonstrate how host apps should integrate. A good example of this approach is [Alchemy CMS](https://github.com/AlchemyCMS/alchemy_cms).
 
-### Hot module replacement
+### Hot Module Replacement (HMR)
 
 Webpacker out-of-the-box supports HMR with webpack-dev-server and you can toggle it by setting dev_server/hmr option inside webpacker.yml.
 
-Check out this guide for more information:
-
-https://webpack.js.org/configuration/dev-server/#devserver-hot
-To support HMR with React you would need to add react-hot-loader. Check out this guide for more information:
+Check out [webpack's documentation on DevServer](https://webpack.js.org/configuration/dev-server/#devserver-hot) for more information.
 
-https://gaearon.github.io/react-hot-loader/getstarted/
+To support HMR with React you would need to add react-hot-loader. Check out [React Hot Loader's _Getting Started_ guide](https://gaearon.github.io/react-hot-loader/getstarted/).
 
-Don't forget to disable HMR if you are not running webpack-dev-server otherwise you will get not found error for stylesheets.
+Don't forget to disable HMR if you are not running webpack-dev-server otherwise you will get "not found error" for stylesheets.
```

Comment:
Not sure where the quotes go here. Possibly:
```suggestion
Don't forget to disable HMR if you are not running webpack-dev-server otherwise you will get "not found" errors for stylesheets.
```